$NetBSD: patch-data_Makefile,v 1.1 2015/08/24 02:04:21 khorben Exp $

Do not install the desktop entry for gdeasm for now (as it is not packaged)

--- data/Makefile.orig	2015-08-24 01:33:54.000000000 +0000
+++ data/Makefile
@@ -16,8 +16,6 @@ install:
 	$(MKDIR) $(DESTDIR)$(PREFIX)/share/applications
 	$(INSTALL) -m 0644 deforaos-coder.desktop $(DESTDIR)$(PREFIX)/share/applications/deforaos-coder.desktop
 	$(MKDIR) $(DESTDIR)$(PREFIX)/share/applications
-	$(INSTALL) -m 0644 deforaos-gdeasm.desktop $(DESTDIR)$(PREFIX)/share/applications/deforaos-gdeasm.desktop
-	$(MKDIR) $(DESTDIR)$(PREFIX)/share/applications
 	$(INSTALL) -m 0644 deforaos-sequel.desktop $(DESTDIR)$(PREFIX)/share/applications/deforaos-sequel.desktop
 	$(MKDIR) $(DESTDIR)$(PREFIX)/share/applications
 	$(INSTALL) -m 0644 deforaos-simulator.desktop $(DESTDIR)$(PREFIX)/share/applications/deforaos-simulator.desktop
